docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -0,0 +1,260 @@
+
+After rewriting the core compute of the tutorial Airflow Dag in Dagster we're left with:
+
+```python
+import time
+from datetime import date, timedelta
+
+from dagster import job, op
+
+@op
+def print_date():
+    dt = date.today()
+    print(dt)
+    return dt
+
+@op
+def sleep(dt: date):
+    time.sleep(5)
+
+@op
+def templated(dt: date):
+    for i in range(5):
+        print(dt)
+        print(dt - timedelta(days=7))
+
+@job
+def tutorial_job():
+    dt = print_date()
+    sleep(dt)
+    templated(dt)
+
+@repository
+def rewrite_repo():
+    return [tutorial_job]
+```
+
+### Defining the schedule
+
+next we'll add a schedule, in dagster unlike airflow schedules are defined seperately from the definition of compute
+
+```python
+schedule = ScheduleDefinition(job=tutorial_job, cron_schedule="@daily")
+
+@repository
+def rewrite_repo():
+    return [tutorial_job, schedule]
+```
+
+At this point we've done a basic rewrite of the tutorial dag but have left out a few features
+
+### Configuration
+
+#### Retries
+
+##### Job Level Retries
+
+Job level retries are managed by the [run launcher](/deployment/run-retries) you will need to enable support for them in your dagster.yaml, once you do you can define the retry count on the job.
+
+```python
+@job(tags={"dagster/max_retries": 1})
+def tutorial_job():
+    dt = print_date()
+    sleep(dt)
+    templated(dt)
+```
+
+##### Op Level Retries
+
+In the tutorial Dag the sleep task allowed for 3 retries, to configure this in dagster you can use [Op level retry policies](/concepts/ops-jobs-graphs/op-retries)
+
+```python
+@op(
+    retry_policy=RetryPolicy(
+        max_retries=3
+    )
+)
+def sleep(dt: date):
+    time.sleep(5)
+```
+
+#### Tagging and Metadata
+
+Dagster has a [tagging and metadata system](/concepts/ops-jobs-graphs/metadata-tags) that is quite a bit more powerful and integrated with the UI than what exists in Airflow.
+
+```python
+@job(tags={"dagster/max_retries": 1, "dag_name": "example"})
+def tutorial_job():
+    dt = print_date()
+    sleep(dt)
+    templated(dt)
+```
+
+### The final product
+
+```python
+import time
+from datetime import date, timedelta
+
+from dagster import RetryPolicy, ScheduleDefinition, job, op, repository
+
+
+@op
+def print_date():
+    dt = date.today()
+    print(dt)
+    return dt
+
+@op(
+    retry_policy=RetryPolicy(
+        max_retries=3
+    )
+)
+def sleep(dt: date):
+    time.sleep(5)
+
+@op
+def templated(dt: date):
+    for i in range(5):
+        print(dt)
+        print(dt - timedelta(days=7))
+
+@job(tags={"dagster/max_retries": 1, "dag_name": "example"})
+def tutorial_job():
+    dt = print_date()
+    sleep(dt)
+    templated(dt)
+
+schedule = ScheduleDefinition(job=tutorial_job, cron_schedule="@daily")
+
+@repository
+def rewrite_repo():
+    return [tutorial_job, schedule]
+```
```

Comment:
I think all of this should be done within the steps, and should not be mentioned separately. Otherwise, we are mixing the purposes of this guide with concept mapping.

Reply:
Of these two options, I would vote to include the concepts in the tutorial, or at least to include retries. Retries are a critical part of Airflow that Prefect also makes a big deal about. The mapping to dagster concepts is not straightforward (op retries vs job retries) so it'd be nice to elaborate in the tutorial.

Reply:
I think to have a baseline for this tutorial, we should only focus mapping the Airflow tutorial to its Dagster equivalent.

Retries at the operator and DAG level are mentioned in the Airflow tutorial, so we should definitely mention them in our tutorial.

But my suggestion is not to go too into depth in the tutorial for what the concept is. We should just link out the Concept Mapping guide, which will explain the parity between Airflow's retries and our retries. I'm thinking at the end of each step in our tutorial, we link a "See also: `retries`, `ops`, ...". There, the user can see the in-depth comparison between the specific concepts.

Otherwise, we risk information dumping so much content onto the new user.